tests/unit/src/test/resources/foldingRange/expect/Try.scala

```diff
@@ -1,41 +1,41 @@
 class A>>region>>{
-  val shortExpr =
+  val shortExpr =>>region>>
     try ???
-    catch>>region>> {
+    catch >>region>>{
       case 0 =>
       case 1 => println()
       case 2 =>>>region>>
         println()
         println()
         println()
-        println()
-<<region<<      case 3 =>
+        println()<<region<<
+      case 3 =>
         println()
       case _ => println()
-    }<<region<<
+    }<<region<<<<region<<
 
-  val aTry =
+  val aTry =>>region>>
     try >>region>>{
       ???
       ???
       ???
       ???
-    }<<region<< catch>>region>> {
+    }<<region<< catch >>region>>{
       case 0 =>
       case 1 => println()
       case 2 =>>>region>>
         println()
         println()
         println()
-        println()
-<<region<<      case 3 =>
+        println()<<region<<
+      case 3 =>
         println()
       case _ => println()
-    }<<region<< finally >>region>>{
+    } finally >>region>>{
       ???
       ???
       ???
       ???
       ???
-    }<<region<<
+    }<<region<<<<region<<<<region<<
```

Comment:
This should be 2 regions, no? One from finally and another from val.

Reply:
fixed with the catch region